osi_common.proto

```diff
@@ -447,6 +546,23 @@ message BaseStationary
     // The polygon is defined counter-clockwise.
     //
     repeated Vector2d base_polygon = 4;
+
+    // Sub-divisions of the overall bounding box of the \c BaseStationary object.
+    //
+    // The bounding box sections can include separate parts on partially-opaque
+    // objects such as trees with a distinction between trunk and crown.
+    // 
+    // \note The bounding box sub-divisions can extend beyond the overall
+    // bounding box, however no actual geometry must reside outside of the
```

Comment:
Except for side mirrors? I somehow don't like this. I am thinking about Praktiker "alles außer Tiernahrung"

Reply:
Yeah! The problem is, that the side mirror is already excluded, so it can't be included in the sub bounding boxes.

Reply:
I think we have to create a TODO here for a major version change to discuss this again. If Sub-Bounding-Boxes would have existed in the beginning this exclusion wouldn't have happend.

Reply:
I would propose not to add TODOs in the documentation but rather create an issue with the breaking change label.